tests/ui/macros/rfc-2011-nicer-assert-messages/assert-with-custom-errors-does-not-create-unnecessary-code.rs

```diff
@@ -1,4 +1,10 @@
-//@ compile-flags: --test
+//@ revisions: can_unwind
+//@ [can_unwind] compile-flags: --test
+//@ [can_unwind] needs-unwind
+//
+//@ revisions: cannot_unwind
+//@ [cannot_unwind] compile-flags: --test -Zpanic_abort_tests
```

Comment:
I think `-Zpanic-abort-tests` does nothing for `-Cpanic=unwind` so enabling it unconditionally should be fine.

Reply:
indeed, that works. updated the PR